test/algorithms/physical_design/graph_oriented_layout_design.cpp

```diff
@@ -103,71 +103,85 @@ TEST_CASE("Different parameters", "[graph-oriented-layout-design]")
 
     graph_oriented_layout_design_params params{};
 
-    // Low effort mode
+    // high-efficiency mode
     params.mode = graph_oriented_layout_design_params::effort_mode::HIGH_EFFICIENCY;
-    // Return first found layout
+    // return first found layout
     params.return_first = true;
     const auto layout1  = graph_oriented_layout_design<gate_layout>(ntk, params, &stats);
 
     REQUIRE(layout1.has_value());
     check_eq(ntk, *layout1);
 
-    // Verbose mode and timeout
+    // verbose mode and timeout
     params.timeout     = 100000;
     params.verbose     = true;
     const auto layout2 = graph_oriented_layout_design<gate_layout>(ntk, params, &stats);
 
     REQUIRE(layout2.has_value());
     check_eq(ntk, *layout2);
 
-    // High effort mode
+    // high-effort mode
     params.mode        = graph_oriented_layout_design_params::effort_mode::HIGH_EFFORT;
     params.verbose     = false;
     const auto layout3 = graph_oriented_layout_design<gate_layout>(ntk, params, &stats);
 
     REQUIRE(layout3.has_value());
     check_eq(ntk, *layout3);
 
-    // Highest effort mode
+    // highest-effort mode
     params.mode        = graph_oriented_layout_design_params::effort_mode::HIGHEST_EFFORT;
     params.verbose     = false;
     const auto layout4 = graph_oriented_layout_design<gate_layout>(ntk, params, &stats);
 
     REQUIRE(layout4.has_value());
     check_eq(ntk, *layout4);
 
-    // Full search
-    params.mode         = graph_oriented_layout_design_params::effort_mode::HIGH_EFFORT;
-    params.return_first = false;
-    const auto layout5  = graph_oriented_layout_design<gate_layout>(ntk, params, &stats);
+    // maximum-effort mode
+    params.mode        = graph_oriented_layout_design_params::effort_mode::MAXIMUM_EFFORT;
+    const auto layout5 = graph_oriented_layout_design<gate_layout>(ntk, params, &stats);
 
```

Comment:
I suggest refactoring these tests into multiple `SECTION`s with the parameters definition outside. That way, it is already reset to default, so that you only need to set the required parameters that differ from the default.